workshops/popup_chrome_extension/README.md

```diff
@@ -0,0 +1,498 @@
+    if (index > -1) {
+      elements.splice(index, 1);
+    }
+  }
+
+}
+```
+
+Now we need functions to popup and copy text:
+
+```javascript
+ let open1 = document.getElementById('open1');
+let open2 = document.getElementById('open2');
+
+function popup1(openQ) {
+    if (!openQ) {
+        document.getElementById("main").style.visibility = "hidden";
+        document.getElementById("add").style.height = "75px";
+        document.getElementById('open1').innerHTML = "(+)";
+    } else {
+        document.getElementById("main").style.visibility = "initial";
+        document.getElementById("add").style.height = "425px";
+        document.getElementById('open1').innerHTML = "(-)";
+    }
+}
+
+function popup2(openQ) {
+    if (!openQ) {
+        document.getElementById("bookmarks").style.visibility = "hidden";
+        document.getElementById('open2').innerHTML = "(+)";
+    } else {
+        document.getElementById("bookmarks").style.visibility = "initial";
+        document.getElementById('open2').innerHTML = "(-)";
+    }
+}
+popup1(false);
+
+function copy(x, z) {
+    let y = "" + x;
+    //console.log(y)
+    chrome.storage.sync.get(y, function(result) {
+        //console.log(result[y])
+        var textArea = document.createElement("textarea");
+
+        if (z == 0) {
+            textArea.value = result[y].meeting_link;
+        } else if (z == 1) {
+            textArea.value = result[y].password;
+        } else if (z == 2) {
+            textArea.value = result[y].password;
+        }
+
+
+        document.body.appendChild(textArea);
+        textArea.select();
+        document.execCommand("Copy");
+        textArea.remove();
+
+        if (z == 0) {
+            if (result[y].password != 0) {
+                alert("MEETING INFO Copied! You can paste it directly into ZOOM.\n\nIMPORTANT: The meeting password will be copied once you click 'OK'");
+                copy(y, 2)
+            } else {
+
+                alert("MEETING LINK Copied! Use CRTL/CMD + V to paste the link in your browser.");
+            }
+        } else if (z == 1) {
+            alert("The Zoom meeting should open once you press OK.\n\nIMPORTANT: The meeting password is copied and you can paste it into the prompt.");
+        } else if (z == 2) {
+            alert("PASSWORD Copied!");
+        }
+
+    });
+
+```
+
+Lastly, we need functions to open the link in a new tab and **wait for buttons to be pressed**. This Github Gist will help you create eventListeners for listening to button presses: https://gist.githubusercontent.com/shreyfirst/0da1748f048b931f8e9ac13223a50c0a/raw/e554eda21c75411713bf2ca5df96d5d7bd67d317/Final%2520code
+
+This goes below everything you've coded so far and above the closing of the eventListener that houses once the page loads. These is a Gist to the final code: https://gist.githubusercontent.com/shreyfirst/d980597b6531fddd8d77cb8e63f6ffb3/raw/0d1b7ddffd7f97810998fe2c93d7a814f889562e/Full%2520code
```

Comment:
Can you add a code block that shows the eventListener and then a comment like `// your code here`, just for more context?